src/xdocs/google_style.xml

```diff
@@ -2302,24 +2302,62 @@
         <p>
           It is possible to suppress some violations by embeded filters
           <a href="config_filters.html#SuppressionFilter">
-            SuppressionFilter</a>
-          and
+            SuppressionFilter</a>,
           <a href="config_filters.html#SuppressionXpathFilter">
-            SuppressionXpathFilter</a>.
+            SuppressionXpathFilter</a>,
+          <a href="config_filters.html#SuppressWithNearbyCommentFilter">
+            SuppressWithNearbyCommentFilter</a>,
+          <a href="config_filters.html#SuppressionCommentFilter">
+            SuppressionCommentFilter</a>
+          and
+          <a href="config_filters.html#SuppressWarningsFilter">
+            SuppressWarningsFilter</a>
+          in conjunction with
+          <a href="config_filters.html#SuppressWarningsHolder">
+            SuppressWarningsHolder</a>.
+        </p>
+        <p>
           Location of config file for <i>SuppressionFilter</i> can be defined by system property
           <i>org.checkstyle.google.suppressionfilter.config</i> (default value is
           <i>checkstyle-suppressions.xml</i>).
+        </p>
+        <p>
           Location of config file for <i>SuppressionXpathFilter</i>
           can be defined by system property
           <i>org.checkstyle.google.suppressionxpathfilter.config</i> (default value is
           <i>checkstyle-xpath-suppressions.xml</i>).
         </p>
+        <p>
+          To suppress a check on a block of code using <i>SuppressWithNearbyCommentFilter</i>
```

Comment:
You meant SuppressionCommentFilter
And SuppressWithNearbyCommentFilter is for next block.